Jenkinsfile

```diff
@@ -10,11 +10,13 @@ properties([
     ]
 ])
 
+def SDK_URL = "master/24.18.0/1.xs8/71/"
```

Comment:
wasn't the plan to have this as a parameter or am I missing something? :thinking:

Reply:
Previous discussion on https://citrix.slack.com/archives/G01LQNPJUV7/p1723109406878149?thread_ts=1723105813.527799&cid=G01LQNPJUV7